ale_linters/python/pyls.vim

```diff
@@ -0,0 +1,21 @@
+" Author: aurieh <me@aurieh.me>
+" Description: A language server for Python
+
+call ale#Set('python_pyls_executable', 'pyls')
```

Comment:
I just noticed one thing missing, could you add a documentation entry for this new option?

Reply:
Oh sure, one second.

Reply:
On that note, is there a vim plugin that helps with doc file alignment? Doing it by hand is kinda confusing.

Reply:
There's a fixer for it. See `:ALEFixSuggest` when editing a help file.

Reply:
Oh that's really cool, thanks for letting me know.